sass/patternfly-react.scss

```diff
@@ -0,0 +1,23 @@
+/**
+  Third Party / Dependency imports here
+*/
+$font-path:      '~patternfly/dist/fonts/';
+$img-path:        '~patternfly/dist/img/';
+$icon-font-path: '~patternfly/dist/fonts/';
+
+
+// Bootstrap Core variables and mixins
+@import "bootstrap/variables";
+@import "bootstrap/mixins";
+
+// Patternfly Core variables and mixins
+@import 'patternfly/variables';
+@import 'patternfly/fonts';
+@import 'patternfly/bootstrap-mixin-overrides';
+@import 'patternfly/mixins';
+
+/**
+  Patternfly React Specific Extensions
+*/
+
+@import 'patternfly-react/patternfly-react';
```

Comment:
Isn't it a bit confusing?
What about:
```scss
// patternfly-react.scss
@import 'partials';

// _partials.scss
@import 'partials/partial1';
@import 'partials/partial2';
...
@import 'partials/partialN';
```

Reply:
I can rename the `patternfly-react` directory to `partials` if that makes it less confusing. The `patternfly-react/patternfly-react` include is the full partials include file (or will be once we have partials for components). I would like that file to remain as `_patternfly-react.scss` so it is an obviously include name for applications.

Reply:
Again, correct me if I am wrong. The goal here is to have _patternfly-react.scss which will contain imports for all individual components scss.
This partial will be used by consuming apps as well as storybook/sass/base.scss (which is an example of consuming app).

In that case this file (sass/patternfly-react.scss) is not necessary as all of its contents should be included in base.scss

Reply:
:+1:

Reply:
agreed, except that it is required to build the css. This file is not included in dist/sass

Reply:
yes - I think I am in agreement w/ @jeff-phillips-18. In order to ship the additional css composed in pf-react as a separate file, it will require importing bootstrap/patternfly scss variables, mixins, etc. into the partial to extend that CSS separately. Do we care to do this @sharvit @jtomasek @jeff-phillips-18 ? I guess it really comes down to if we plan to consume that as CSS...

Reply:
+1 I am fine with producing compiled css in addition to sass partial. It definitely does not hurt anything and lets consumer app use react-patternfly with compiled patternfly.css.
Regarding less support, we can add this in subsequent patch if needed. It is currently not an issue as _patternfly_react.scss does not include anything.

Reply:
@jeff-phillips-18 It would be nice to include comment in sass/patternfly-react.scss describing that it is used to compile css or even rename it. It is slightly confusing as it is not entirely clear that base.less imports _patternfly-react.scss partial rather than sass/patternfly-react.scss without looking at webpack.config.js